### PR TITLE
chore: harness fixes for slurm

### PR DESCRIPTION
**Pull request by stoksc**
## Description
Just a few minor tweaks for determined under slurm, mostly debugging aid.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: harness/determined/launch/horovod.py [hunk 1]
```diff
@@ -172,9 +179,22 @@ def main(hvd_args: List[str], script: List[str], autohorovod: bool) -> int:
     worker_wrapper_cmd = create_worker_wrapper_cmd(info.allocation_id)
 
     logging.debug(f"chief worker calling horovodrun with args: {hvd_cmd[1:]} ...")
-
     os.environ["USE_HOROVOD"] = "1"
 
+    if os.environ.get("DET_RESOURCES_TYPE") == constants.RESOURCES_TYPE_SLURM_JOB:
+        # We now have environment images with built-in OpenMPI. When invoked the
```
Comment (stoksc) [inline harness/determined/launch/horovod.py:185]: cc @jerryharrow

Comment (stoksc) [reply harness/determined/launch/horovod.py:185]: if you want to just take this an reopening as yourself that is fine.

**Comment by rb-determined-ai** [comment]
@stoksc what's going on with this PR?  I'm sorry, you probably told me and I forgot.

**Comment by stoksc** [comment]
on hold til next week at least

**Pull request closed without merging**